packages/interbit-ui-components/src/components/ReduxForm/validation.js

```diff
@@ -0,0 +1,58 @@
+const email = value => {
+  const error = 'Invalid email address.'
+  if (
+    /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(value) ||
+    value === undefined ||
+    value === null ||
+    value === ''
+  ) {
```

Comment:
This caught me out. By this definition an undefined email address is a valid email address, which seems strange to me. Granted this can be combined with `required` but if you are asking for an email address, it is probably required anyway. Perhaps separate tests, `requiredEmail` and `optionalEmail` with the appropriate handling of undefined would be clearer.